### ISSUE TITLE
Use latest typescript compiler version

### ISSUE THREAD

**Issue by claudio-silva** [timjroberts/typescript-atomizer#12]
Hi,
I tried to edit a project I wrote in typescript that makes use of the latest (version 1.4.1) language features (like template strings, union types, etc).
I get error warnings from the editor, and wrong type signatures on the tooltips, so I assume this package is using an older typescript compiler/language service.
Could you either:
- update it to the latest typescript version, or
- better yet, make the package make use of the compiler already installed on the user's computer? That way your package would always be up to date concerning typescript feature support. (There could be also  a configurable setting that allowed the user to choose between the bundled and the external compiler).

**Comment by timjroberts** [comment]
Yes, currently, the released 1.3 language services are baked in. I've started looking at ways in which the different versions of the TypeScript language services can work as plug ins, and as you say allow configuration to indicate which version of those language services that you want for the project.

**Comment by claudio-silva** [comment]
Thanks for considering this issue.

I'm not sure this tip will be useful to you, but still:
Os OS X, the `tsc` compiler is installed on `/usr/local/lib/node_modules/typescript/bin/`.
There, besides the compiler, you'll also find the language services:

```
-rw-r--r--   1 nobody  staff    49K Jan 16 17:28 lib.core.d.ts
-rw-r--r--   1 nobody  staff   207K Jan 16 17:28 lib.core.es6.d.ts
-rw-r--r--   1 nobody  staff   584K Jan 16 17:28 lib.d.ts
-rw-r--r--   1 nobody  staff   535K Jan 16 17:20 lib.dom.d.ts
-rw-r--r--   1 nobody  staff   718K Jan 16 17:28 lib.es6.d.ts
-rw-r--r--   1 nobody  staff   1.3K Jan 16 17:20 lib.scriptHost.d.ts
-rw-r--r--   1 nobody  staff    58K Jan 16 17:20 lib.webworker.d.ts
-rwxr-xr-x   1 nobody  staff    40B Jan 16 17:20 tsc
-rw-r--r--   1 nobody  staff   965K Jan 16 17:28 tsc.js
-rw-r--r--   1 nobody  staff    66K Jan 16 17:28 typescript.d.ts
-rw-r--r--   1 nobody  staff    66K Jan 16 17:28 typescriptServices.d.ts
-rw-r--r--   1 nobody  staff   1.4M Jan 16 17:28 typescriptServices.js
-rw-r--r--   1 nobody  staff    15K Jan 16 17:28 typescriptServices_internal.d.ts
-rw-r--r--   1 nobody  staff    15K Jan 16 17:28 typescript_internal.d.ts
```

Perhaps your package could load them from there, if that folder exists?
